docsite/docusaurus.config.ts

```diff
@@ -86,6 +92,15 @@ const config: Config = {
             },
           ],
         },
+        {
+          title: 'Community',
+          items: [
+            {
+              label: 'Tutorial',
+              to: '/docs/community',
+            },
+          ],
+        },
         {
           title: 'Community',
           items: [
```

Comment:
Given that this link doesn't actually work (because I didn't realize I needed a particular invite link) - just remove this and the section ? Or fix the link in here to be the same as in the header ? Or not add the one in the header at all and only here ? Also, any additional items we want to throw under the title "Community" we should put in the items collection, and not add any other sections titled Community, that's just confusing. Typically, you'd put discord/twitter/whatever else you got here.

Reply:
I'm trying to make the changes for the community link and had a couple of questions. I have removed the extra community from the footer and inserted a invite link to the community that was there before. If someone were to click on it, it takes them to our discord but it says that I am personally inviting them so it shows my name and user. Not sure if there is a better link we should be using. Also, you mentioned not having the community in the header at all or making sure both communities (from header and footer) are the same. Which would you suggest is the better option?

Reply:
I don't know that my opinion here is totally relevant, but if you put yourself in the shoes of a person coming to read this page - would you expect to have two links to the same spot, or would that confuse you ? I think a person looking for a Discord link should find it.

No idea about your first question, with making the invite link not personal to you. Looks like this one works (may require a paid account though ?): https://support.discord.com/hc/en-us/articles/115001542132-Custom-Invite-Link

Reply:
I removed the community from the nav bar and decided to just keep the one in the footer. I also changed the link for the discord to never expire.